mesa-quilt

```diff
@@ -18,17 +18,73 @@
 # 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
 d=. ; while [ ! -d "$d/nx-X11/extras" ] && [ "$(readlink -e "$d")" != "/" ]; do d="$d/.."; done
```

Comment:
Can you explain why we need this? We have two cases: 
1. we are in the base directory. Then the path to mesa is clear
2. we are not in the base directory. Then we can use $(dirname $0) to get to defined place

I don't see why climbing up until / gets hit makes any sense...